treeherder/client/thclient/client.py

```diff
@@ -349,6 +349,17 @@ def init_data(self):
             }
 
 
+class TreeherderPossibleJob(TreeherderData, ValidatorMixin):
```

Comment:
Thinking about this more, I'm not sure we need to enable this in the ``thclient``.  This is really just an internal process.  We aren't going to have external folks post this data to TH, are we?  If so, then cool as it is.  But if not, I think we could just have the celery task write directly to the DB (through ``jobs.py`` as you're doing) without going through a call to an API.